db/schema.rb

```diff
@@ -433,7 +433,7 @@
     t.string "preferred_language", default: "tr"
     t.datetime "created_at", null: false
     t.datetime "updated_at", null: false
-    t.datetime "password_changed_at", default: -> { "now()" }, null: false
+    t.datetime "password_changed_at", default: -> { "CURRENT_TIMESTAMP" }, null: false
```

Comment:
Şunu bi türlü standarta bağlayamadık :smile:

Her 5 PR'dan birinde kesin ya `now()` oluyor, yada `CURRENT_TIMESTAMP`.

Reply:
Bunun sebebi, tahminimce, farklı Postgres sürümleri kullanıyor olmamız. Benim sürüm 10.5, sen de bi kontrol eder misin?

Reply:
İrfan Vagrant ile çalışıyor, orada (omu/debian-stable-server imajı) Debian stretch ile gelen [postgres](https://packages.debian.org/stretch/postgresql) kurulu olmalı, yani 9.6.  @sinansh sonuç: geliştirme ortamları farklı; ayrıca bu önemsiz bir farklılık.

Reply:
@sinansh "nitpicking in code reviews" şeklinde arat ;-)  bundan kaçınalım